src/main/java/org/cascadebot/cascadebot/commandmeta/CommandContext.java

```diff
@@ -297,8 +297,13 @@ public String globalEmote(String key) {
     }
 
     public boolean hasPermission(String permission) {
+        return hasPermission(member, permission);
+    }
+
+    public boolean hasPermission(Member member, String permission) {
         CascadePermission cascadePermission = CascadeBot.INS.getPermissionsManager().getPermission(permission);
-        return cascadePermission != null; // TODO: Check actual perms
+        if (cascadePermission == null) return false;
```

Comment:
we should maybe log this or throw an exception as this should only happen when they send in the wrong permission string